### PR TITLE
memberlist tag

### PR DESCRIPTION
**Pull request by intoeetive**
make `exp:member:memberlist` tag use `role` instead of `member_group`

EECORE-918

Docs PR: https://github.com/ExpressionEngine/ExpressionEngine-User-Guide/pull/332

### CODE REVIEW

File: system/ee/ExpressionEngine/Addons/member/mod.member_memberlist.php [hunk 1]
```diff
@@ -327,7 +329,7 @@ public function memberlist($is_search_form = false)
             // Normalizing cp available sorts
             $order_by = ($order_by == 'username') ? 'screen_name' : $order_by;
             $order_by = ($order_by == 'dates') ? 'join_date' : $order_by;
-            $order_by = ($order_by == 'role') ? 'role_id' : $order_by;
+            $order_by = ($order_by == 'member_group') ? 'role_id' : $order_by;
```
Comment (TomJaeger) [inline system/ee/ExpressionEngine/Addons/member/mod.member_memberlist.php]: @intoeetive should this be member_group?  or Role?

**Review by TomJaeger — COMMENTED** [review]
Have one question for you here... https://github.com/ExpressionEngine/ExpressionEngine/pull/1617/files#r824896203

Otherwise it looks good to me.

Also looks like there's a conflict on the tests

File: system/ee/ExpressionEngine/Addons/member/mod.member_memberlist.php (hunk 1 above)
Comment (intoeetive) [reply system/ee/ExpressionEngine/Addons/member/mod.member_memberlist.php]: It should actually be both. Fixed

**Review by bryannielsen — APPROVED** [review]
Adding formal approval

**Pull request merged**